RealmModelGenerator/NibDesignable.swift

```diff
@@ -68,18 +69,19 @@ extension NSView {
      - returns: Name of a single view nib file.
      */
     public func nibName() -> String {
-        return self.dynamicType.description().componentsSeparatedByString(".").last!
+        return String(type(of: self).description().split(separator: ".").last!)//.componentsSeparatedByString(".").last!
```

Comment:
go ahead and remove comment

Reply:
How do I make this modification. 
Do I add another commit or what?

Reply:
you would make another commit saying that you removed a comment